test/sdk.test.ts

```diff
@@ -2,80 +2,50 @@ import {
   extractErrMsg,
   initRune,
   runePostMessageHandler,
+  sendRuneAppCommand,
   simulateNativeApp,
 } from "./helper"
-import { getRuneSdk, RuneExport } from "../src"
+import { getRuneSdk } from "../src/sdk"
 
-let Rune: RuneExport
+const CHALLENGE_NUMBER = 123
 
 beforeEach(async () => {
   simulateNativeApp()
-  Rune = getRuneSdk()
 })
 
 describe("sdk", function () {
-  test("init() -> startGame() -> pauseGame() -> resumeGame()", async function () {
-    const Rune = getRuneSdk()
-    // Mock a game's state and mimic running inside Rune where env is set
-    let gameState: "WAITING" | "RUNNING" | "PAUSED" = "WAITING"
-    initRune(Rune, {
-      startGame: () => {
-        gameState = "RUNNING"
-      },
-      pauseGame: () => {
-        gameState = "PAUSED"
-      },
-      resumeGame: () => {
-        gameState = "RUNNING"
-      },
-      getScore: () => {
-        return 0
-      },
-    })
-
-    // Should be no change in gameState from calling init()
-    expect(gameState).toMatchInlineSnapshot(`"WAITING"`)
-
-    // Should start the game
-    Rune._startGame()
-    expect(gameState).toMatchInlineSnapshot(`"RUNNING"`)
-
-    // Should pause the game
-    Rune._pauseGame()
-    expect(gameState).toMatchInlineSnapshot(`"PAUSED"`)
-
-    // Should resume the game
-    Rune._resumeGame()
-    expect(gameState).toMatchInlineSnapshot(`"RUNNING"`)
-  })
-
-  test("don't allow calling other functions before init()", async function () {
-    expect(
-      await extractErrMsg(() => {
-        Rune._startGame()
-      })
-    ).toMatchInlineSnapshot(`"Rune._startGame() called before Rune.init()"`)
-  })
-
   test("ensure correct properties passed to init()", async function () {
+    const { Rune } = getRuneSdk(1)
```

Comment:
Could consider also using `CHALLENGE_NUMBER` here to avoid magic constant `1` :)